packages/playground/blueprints/src/lib/steps/activate-plugin.ts

```diff
@@ -51,8 +51,7 @@ foreach ( ( glob( $plugin_path . '/*.php' ) ?: array() ) as $file ) {
 echo 'NO_ENTRY_FILE';
 `,
 	});
-	if (result.errors) throw new Error(result.errors);
-	if (result.text === 'NO_ENTRY_FILE') {
+	if (result.text.endsWith('NO_ENTRY_FILE')) {
```

Comment:
Some plugins trigger warnings during the installation – let's check just the last part of the output text